### PR TITLE
WorkAround #2179

### PR DESCRIPTION
**Pull request by j0rt3g4**
WorkAround #2179.

Basically, this is a technical issue that I've recreated in a virtual environment.
I consider this as a bug, so on the documentation itself, there's nothing related to this issue that should be added without a full revision from an expert on Microsoft.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-exploit-guard/customize-exploit-protection.md [hunk 1]
```diff
@@ -252,6 +254,8 @@ Set-ProcessMitigation -Name processName.exe -Enable EnableExportAddressFilterPlu
 
 See the [Windows Security](../windows-defender-security-center/windows-defender-security-center.md#customize-notifications-from-the-windows-defender-security-center) topic for more information about customizing the notification when a rule is triggered and blocks an app or file.
 
+ 
+
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-exploit-guard/customize-exploit-protection.md]: These 2 empty blank lines are redundant. Please remove.

File: windows/security/threat-protection/windows-defender-exploit-guard/customize-exploit-protection.md [hunk 2]
```diff
@@ -100,6 +100,8 @@ Validate stack integrity (StackPivot) | Ensures that the stack has not been redi
 >The result will be that DEP will be enabled for *test.exe*. DEP will not be enabled for any other app, including *miles.exe*. 
 >CFG will be enabled for *miles.exe*.
 
+>[!NOTE] If you found any issues on this article you can report it directly to a Windows Server/Windows Client partner or use our technical support numbers on your country.
+
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-exploit-guard/customize-exploit-protection.md]: If the added text will be accepted, you should probably rephrase the following:
> ... or use **the Microsoft** technical support numbers **for** your country.

Comment (illfated) [reply windows/security/threat-protection/windows-defender-exploit-guard/customize-exploit-protection.md]: Also, code style adjustment needed: add a NewLine after `>[!NOTE]` as used in the other notes, warnings and important notices elsewhere in the docs.
```
>[!NOTE]
>If you found any issues on this article, you can report it directly to a Windows Server/Windows Client partner or use the Microsoft technical support numbers for your country.
```

Comment (JohanFreelancer9) [reply windows/security/threat-protection/windows-defender-exploit-guard/customize-exploit-protection.md]: Agree with copy as edited by illfated.

File: windows/security/threat-protection/windows-defender-exploit-guard/customize-exploit-protection.md [hunk 3]
```diff
@@ -100,6 +100,8 @@ Validate stack integrity (StackPivot) | Ensures that the stack has not been redi
 >The result will be that DEP will be enabled for *test.exe*. DEP will not be enabled for any other app, including *miles.exe*. 
 >CFG will be enabled for *miles.exe*.
 
+>[!NOTE] If you found any issues on this article you can report it directly to a Windows Server/Windows Client partner or use our technical support numbers on your country.
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-exploit-guard/customize-exploit-protection.md]: Just to make it even more visible:
```suggestion
>[!NOTE]
>If you have found any issues in this article, you can report it directly to a Windows Server/Windows Client partner or use the Microsoft technical support numbers for your country.
```

File: windows/security/threat-protection/windows-defender-exploit-guard/customize-exploit-protection.md (hunk 1 above)
Comment (j0rt3g4) [reply windows/security/threat-protection/windows-defender-exploit-guard/customize-exploit-protection.md]: Done

**Comment by mypil** [comment]
@andreabichsel - can you please assist to check, approve and merge the content for this PR based on issue #2179?

Thank you.

**Comment by illfated** [comment]
@j0rt3g4 : Thank you for adding the corrections. The 2 redundant blank lines are still present, though. (Maybe there is a missing commit you have not pushed to Github yet?)

**Comment by j0rt3g4** [comment]
OK I'm gonna check it what lines numbers

**Comment by illfated** [comment]
258 + 259 seem to be the redundant line numbers in your current PR.

**Comment by mypil** [comment]
@j0rt3g4 - please see comment from illfated on the redundant lines and do the necessary changes to this PR. Thank you.

**Comment by j0rt3g4** [comment]
@mypil  (done).

**Comment by mypil** [comment]
@andreabichsel - Can you please assist to check, approve and merge the content for this PR based on issue #2179?

Thank you.

**Comment by mypil** [comment]
@AndreaBarr - would you be able to merge the content of this PR based on issue #2179? Thank you.

**Pull request merged**